src/main/java/com/github/psambit9791/jdsp/transform/DiscreteFourier.java

```diff
@@ -60,61 +64,36 @@ public void dft() {
     // About plotting, please refer here: https://stackoverflow.com/a/25735274
 
     /**
-     * Returns the absolute value of the discrete fourier transformed sequence
+     * Returns the absolute value (amplitude) of the discrete fourier transformed sequence
      * @param onlyPositive Set to True if non-mirrored output is required
      * @throws java.lang.ExceptionInInitializerError if called before executing dft() method
      * @return double[] The absolute DFT output
      */
     public double[] returnAbsolute(boolean onlyPositive) throws ExceptionInInitializerError{
```

Comment:
It would make sense to change this function to returnMagnitue() since we are dealing with magnitudes and phases here.

Reply:
Would indeed make more sense to me; it's just a choice of whether you want continuity with previous versions of jDSP or want the next release to have terminology that makes more sense

Reply:
Release is still in v0.x so making a backward incompatible change will not be a problem.